### ISSUE TITLE
print also the `cause` for errors if any

### ISSUE THREAD

**Issue by darkyzhou** [vitest-dev/vitest#1203]
### Clear and concise description of the problem

There is a new property for `Error` instances called `cause`, see [here](https://v8.dev/features/error-cause) for details.

Currently vitest only prints the outer `Error` instance like this:

![Screen Shot 2022-04-28 at 17 03 53](https://user-images.githubusercontent.com/7220778/165717829-796eea64-9751-4de2-8255-47f7f54c1c58.png)

Since the `BuildError` here is just a wrapper, we can't actually tell where the original `Error` is thrown as well as what the stack trace is.

It would be much helpful for debugging if vitest can also print its inner `cause ` instance, just like [how JVM does](https://stackoverflow.com/questions/5800629/setting-an-exception-cause-in-java)(check out that `Caused by` part):

```
Exception in thread "main" java.lang.RuntimeException: Some other message
    at Exceptions.main(Exceptions.java:4)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: java.lang.RuntimeException: Some message
    at Exceptions.main(Exceptions.java:3)
    ... 5 more
```

### Suggested solution

See above

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by userquin** [comment]
@darkyzhou it seems to be resolved in this PR https://github.com/vitest-dev/vitest/pull/1202

**Comment by sheremet-va** [comment]
<img width="662" alt="Screenshot 2022-04-28 at 12 32 06" src="https://user-images.githubusercontent.com/16173870/165723237-4ef59a39-0024-4884-95ae-29deacb3e9bb.png">

Yep, resolved in #1202

**Comment by darkyzhou** [comment]
thank you
a nice coincidence
